.coderabbit.yaml

```
@@ -0,0 +1,27 @@
+# .coderabbit.yaml
+language: "ko-KR" # 한국어
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)